clang/test/Analysis/scan-build/silence-core-checkers.test

```diff
@@ -1,5 +1,4 @@
-// FIXME: Actually, "perl".
```

Comment:
Instead of `REQUIRES: perl` in (almost?) every scan-build/ test, update
`clang/test/Analysis/scan-build/lit.local.cfg` instead?

There are some `if ... not config.xxx: config.unsupported = True` examples in other `lit.local.cfg` files.